docs/ref/django-admin.txt

```diff
@@ -364,6 +364,21 @@ standard output.
 When this option is set and ``--verbosity`` is greater than 0 (the default), a
 progress bar is shown in the terminal.
 
+Fixtures compression
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
I don't think this needs a new title?

Reply:
I wanted to replicate here the "Compressed fixtures" section in the `loaddata` command.

https://github.com/django/django/blob/master/docs/ref/django-admin.txt#L599

Reply:
Okay fair enough!